test/e2e/pvutil.go

```diff
@@ -62,19 +62,6 @@ type persistentVolumeConfig struct {
 	namePrefix string
 }
 
-// Delete the nfs-server pod. Only done once per KubeDescription().
```

Comment:
See below re: provider segregation.

As the primary persistent_volumes.go file will exclusively test against an NFS volume, it made more sense to keep NFS util code with those tests.